test/runtime/Policies.go

```diff
@@ -1478,3 +1479,125 @@ var _ = Describe("RuntimeValidatedPolicyImportTests", func() {
 		Expect(res.Output().String()).Should(ContainSubstring(allowedVerdict), "Policy trace did not contain %s", allowedVerdict)
 	})
 })
+
+var _ = Describe("RuntimeValidatedPolicyDropAllTests", func() {
+	var (
+		once             sync.Once
+		logger           *logrus.Entry
+		vm               *helpers.SSHMeta
+		dropAllContainer string
+		monitorStop      func() error
+	)
+
+	initialize := func() {
+		logger = log.WithFields(logrus.Fields{"test": "RuntimeValidatedPolicyDropAllTests"})
+		logger.Info("Starting")
+		vm = helpers.CreateNewRuntimeHelper(helpers.Runtime, logger)
+		dropAllContainer = "dropAllContainer"
+	}
+
+	BeforeAll(func() {
+		once.Do(initialize)
+		res := vm.SetPolicyEnforcement(helpers.PolicyEnforcementDefault)
+		res.ExpectSuccess("Setting policyEnforcement to default")
+		areEndpointsReady := vm.WaitEndpointsReady()
+		Expect(areEndpointsReady).Should(BeTrue())
+	})
+
+	JustBeforeEach(func() {
+		monitorStop = vm.MonitorStart()
+	})
+
+	JustAfterEach(func() {
+		vm.ValidateNoErrorsOnLogs(CurrentGinkgoTestDescription().Duration)
+		Expect(monitorStop()).To(BeNil(), "cannot stop monitor command")
+	})
+
+	AfterFailed(func() {
+		vm.ReportFailed()
+	})
+
+	AfterEach(func() {
+		vm.ContainerRm(dropAllContainer).ExpectSuccess("Container dropAllContainer cannot be deleted")
+	})
+
+	Context("DROP_ALL Ingress Policy test", func() {
+
+		BeforeEach(func() {
+			By("Create an endpoint with no labels to test DROP_ALL")
+			res := vm.ContainerCreate(dropAllContainer, "tgraf/netperf", helpers.CiliumDockerNetwork, "", "")
+			res.ExpectSuccess("Failed to create containter with no labels.")
+		})
+
+		It("DROP_ALL Ingress Policy test", func() {
+			By("Starting cilium monitor in background to filter DROP requests")
+			ctx, cancel := context.WithCancel(context.Background())
+			dropRes := vm.ExecContext(ctx, "cilium monitor --type drop")
+			defer cancel()
+
+			endpoints, err := vm.GetAllEndpointsIds()
+			Expect(err).Should(BeNil())
```

Comment:
assert message

Reply:
Please give me an example. `BeNil()` is an assert and you have used this throughout Policies.go. In fact this is simply a copy of the `RuntimeValidatedPolicies`
See `It("L3/L4 Checks", func() {` . 

I would like to focus on the main test first and not nits at this point. Thanks!

Reply:
@manalibhutiyani yes, there is a lot of places that there is no AssertMessages, it has been a rule that in the new code we added the assert message.

Reply:
Done. Added assert message
```
Expect(err).Should(BeNil(), "Unable to get IDs of endpoints")
```